enviro/config_template.py

```diff
@@ -50,4 +50,8 @@
 auto_water = False
 moisture_target_a = 50
 moisture_target_b = 50
-moisture_target_c = 50
\ No newline at end of file
+moisture_target_c = 50
+
+# compensate for usb power
+usb_power = False
```

Comment:
`usb_power` should not be in config.py. See comments within PR

Reply:
Yeah, that makes sense. This started out with the intent that the user would set this, but then I wanted to see if I could change it automatically :slightly_smiling_face: